training/distributed_training/pytorch/model_parallel/gpt2/submit-train-gpt-simple.ipynb

```diff
@@ -0,0 +1,683 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    base_job_name = f'smp-{model_config}-{machine_str}-tp{tp_degree}-pp{pp_degree}-bs{hyperparameters["train_batch_size"]}'
</pre><p>Add a print statement so it's clear what the base job name is from concatenating all these different values?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3034/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>